include/node_api.h

```diff
@@ -0,0 +1,657 @@
+#ifndef SRC_NODE_API_H_
```

Comment:
How about downloading this from Node.js?

Reply:
Nay, IMHO it shall be included in our VSC to prevent unintended changes.

Reply:
I just think to sync these headers are boring work to anyone, to fix the unintended issue, we could lock the tag or commit id, only maintaining this a ID looks really simple than source code :)